osmaxx/excerptexport/migrations/0033_add_countries_as_public_excerpts_20160518_1401.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-05-18 12:01
+from __future__ import unicode_literals
+
+import os
+
+from django.db import migrations
+
+
+def get_polyfile_name_to_file_mapping():
+    from osmaxx.excerptexport._settings import POLYFILE_LOCATION
+    from osmaxx.utils.polyfile_helpers import _is_polyfile
+    filenames = os.listdir(POLYFILE_LOCATION)
+    return {
+        _extract_country_name_from_polyfile_name(filename): filename
+        for filename in filenames if _is_polyfile(filename)
+    }
+
+
+def _extract_country_name_from_polyfile_name(filename):
+    from osmaxx.utils.polyfile_helpers import POLYFILE_FILENAME_EXTENSION
+    name, _ = filename.split(POLYFILE_FILENAME_EXTENSION)
+    return name
+
+
+def merge_countries(apps, schema_editor):  # noqa
+    Excerpt = apps.get_model("excerptexport", "Excerpt")  # noqa
+    countries = Excerpt.objects.filter(country__isnull=False).order_by('name')
+    import itertools
+    for key, group in itertools.groupby(countries, lambda x: x.name):
+        group_list = list(group)
+        if len(group_list) > 1:
+            country_excerpt = group_list[0]
+            for item in group_list[1:]:
```

Comment:
The `item`s here are duplicate country excerpts, aren't they? That's kinda hard to figure out, though. Maybe rename `item` to `duplicate_country_excerpt`.